docs/production.md

```diff
@@ -0,0 +1,155 @@
+# Running an instance of the Research Software Directory in production
+
+## Temporarily disabling the admin interface
+
+You can stop users from making changes to the database by disabling the authentication to the admin interface, as follows:
+
+```shell
+docker-compose stop auth
+```
+
+This can be useful when upgrading the data or software to a newer version.
+
+Note that even with a stopped `auth` service, determined users can still access `backend` directly, and users who were
+logged in before you disable `auth` will still be able to use the admin interface to make changes.
+
+It it recommended that you post a message that users see when they try to use the admin interface. The easiest way to do
+this is as follows:
+
+```shell
+# check volume names
+docker volume ls
+
+# pick the admin interface volume (probably rsd_static_admin)
+docker volume inspect rsd_static_admin
+
+# check where it is mounted
+sudo -i
+cd <to the where the admin static files are>
+
+# rename index.html
+mv index.html index.html.disabled
+
+# put a message in index.html
+echo "<html>" \
+"<head></head>" \
+"<body>Sorry, we're doing maintenance right now. " \
+"Hopefully be back soon.</body></html>" > index.html
+```
+
+You can check if it works by using your browser to navigate to the admin interface. Instead of the normal interface, you
+should now see your message.
+
+
+## Updating a production instance
+
+Every now and then, the production instance needs to be updated, so the server
+can get the latest security patches, and the Research Software Directory
+software itself can be updated to include the latest features.
+
+The steps below differentiate between the old and the new instance of the Research
+Software Directory; the old instance has IP ``35.156.38.208``, the new one has
+IP ``3.122.233.225``. Your IP addresses will likely be different.
+
+1. Make a new Amazon instance by following the notes above. Some things to think about:
+    - Reuse the existing security group.
+    - Reuse the existing key pair.
+    - Verify that you're allowed to ssh into the new instance.
+1. Transfer the ``rsd-secrets.env`` file from the old instance to the new instance.
+
+    ```shell
+    cd $(mktemp -d)
+    scp -i ~/.ssh/rsd-instance-for-nlesc-on-aws.pem \
+       ubuntu@35.156.38.208:/home/ubuntu/rsd/rsd-secrets.env .
+    scp -i ~/.ssh/rsd-instance-for-nlesc-on-aws.pem \
+       ./rsd-secrets.env \
+       ubuntu@3.122.233.225:/home/ubuntu/rsd/rsd-secrets.env
+    ```
+1. On the remote, create the symlink `.env` and let it point to `rsd-secrets.env`:
+
+    ```shell
+    cd ~/rsd
+    ln -s rsd-secrets.env .env
+    ```
+
+1. Stop new additions to the database in the old research software
+   directory instance by stopping the ``rsd-admin`` service.
+
+    ```shell
+    ssh -i ~/.ssh/rsd-instance-for-nlesc-on-aws.pem ubuntu@35.156.38.208
+    cd /home/ubuntu/rsd
+    docker-compose stop rsd-admin
+    ```
```

Comment:
This step outdated and should be replaced with pointers to the ` Temporarily disabling the admin interface` chapter.

Reply:
added